tests/integration_tests/data_steward/retraction/retract_deactivated_pids_test.py

```diff
@@ -0,0 +1,182 @@
+#Python imports
+import os
+import unittest
+import logging
+
+# Third party imports
+import app_identity
+import pandas as pd
+import mock
+
+# Project Imports
+import bq_utils
+from utils.bq import get_client
+import gcs_utils
+from tests import test_util
+from retraction import retract_deactivated_pids
+from io import open
+from constants.cdr_cleaner import clean_cdr as clean_consts
+
+TABLE_ROWS_QUERY = 'SELECT * FROM {dataset_id}.__TABLES__ '
+
+EXPECTED_ROWS_QUERY = ('SELECT * '
+                       'FROM {dataset_id}.{table_id} '
+                       'WHERE person_id = {pid} '
+                       'AND {date_column} >= (SELECT deactivated_date '
+                       'FROM {dataset_id}.{pid_table_id} '
+                       'WHERE person_id = {pid})')
```

Comment:
This query looks like you are checking to make sure data that isn't supposed to be there exists.  That's good.  How about a secondary query to make sure data exists for a participant that should exist.  For example, some data exists after deactivation retraction that should be there.